_schemaV2.graphql

```diff
@@ -8622,18 +8622,15 @@ type CreateVerifiedRepresentativeSuccess {
   verifiedRepresentative: VerifiedRepresentative
 }
 
-# Autogenerated input type of CreateViewingRoom
 input CreateViewingRoomInput {
   attributes: ViewingRoomAttributes
 
   # Main text
   body: String
-
-  # A unique identifier for the client performing the mutation.
```

Comment:
a bunch of changes like that - I've decided to not add "obvious" rails-generated descriptions, but I can do it if you think I should